tests/integration/scenarios.py

```diff
@@ -134,6 +134,48 @@ def test_remove_virtual_interfaces(self):
         res = subprocess.run(['ip', 'link', 'show', 'dev', 'br54'], capture_output=True, text=True)
         self.assertIn('not exist', res.stderr)
 
+    # https://bugs.launchpad.net/netplan/+bug/2083029
+    def test_netplan_try(self):
```

Comment:
nitpick: 
```suggestion
    def test_netplan_try_lp2083029(self):
```

praise: Thank you so much for adding proper integration tests!